API-Examples/2024-11-01/erp_abrufen/10_response_taskGet.xml

```diff
@@ -153,6 +163,19 @@ HTTP/1.1 200 OK
                         </Device>
                     </resource>
                 </entry>
+                <entry>
+                    <fullUrl value="urn:uuid:47decbcb-a411-4298-8b61-eab2374c818a"/>
```

Comment:
Hier ist absolute und relative Referenzierung vermischt.
Am Besten du holst dir via Docker/Titus mal ein RU "Original" und passt das an. Ich glaube aktuell ist überall im Quittungs Bundle absolute Referenzierung

Reply:
Das sieht aus wie das Beispiel, das ich auch in Jenkins erhalten habe. Die Tests wurden gegen RU ausgeführt.

Reply:
Schau dir mal die Datei an: [get_task_id_receipt.txt](https://github.com/user-attachments/files/16189242/get_task_id_receipt.txt). Die habe ich aus der RU. Dort ist im Quittungsbundle alles mit absoluter urn:uuid Referenzierung. Also Z. 147 müsste dann auch absolut sein.

Reply:
Danke! Habe die Referenzen ungebaut